tamr_unify_client/operation.py

```diff
@@ -18,6 +18,21 @@ class Operation(BaseResource):
     def from_json(cls, client, resource_json, api_path=None):
         return super().from_data(client, resource_json, api_path)
 
+    @classmethod
+    def from_resource_id(cls, client, resource_id):
+        """Get an operation by resource ID.
+
+        :param client: Delegate underlying API calls to this client.
```

Comment:
The docstring and typehinting are not in compliance with the current standards.  I ran into difficulties trying to refer to the `Client` and `Operation` classes/types.  I'm not sure if there is a circular import issue or something like that.

Reply:
Type annotations for TUC are best-effort and not required, so this is ok.